<ui version="4.0" >
 <class>Form_hotplugactions</class>
 <widget class="QWidget" name="Form_hotplugactions" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>420</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>370</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="pixmap" >
    <pixmap>../../../../../usr/share/kademar/utils/kademarcenter/img/bg/accions.png</pixmap>
   </property>
   <property name="scaledContents" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="boto_fesaccio" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>310</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>D'Accord</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boto_sortir" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>310</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>CancelÂ·la</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ch_save" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>280</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Recordar siempre para este tipo</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>70</y>
     <width>301</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="iconamedi" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="pixmap" >
    <pixmap/>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>10</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Nou Medi Inserit</string>
   </property>
   <property name="wordWrap" >
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
